### PR TITLE
simplify controller context

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/controller/resource/tccpnoutputs/create.go [hunk 1]
```diff
@@ -72,39 +66,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
...
 
 	{
-		v, err := cloudFormation.GetOutputValue(outputs, DockerVolumeResourceNameKey)
-		if err != nil {
-			return microerror.Mask(err)
-		}
-		cc.Status.TenantCluster.TCCPN.DockerVolumeResourceName = v
-	}
-
-	if r.route53Enabled {
```
Comment (calvix) [inline service/controller/resource/tccpnoutputs/create.go]: why is this dropped?

File: service/controller/resource/tccpn/create.go [hunk 2]
```diff
@@ -322,6 +322,7 @@ func newSecurityGroupsParams(ctx context.Context, cr infrastructurev1alpha2.AWSC
 			MasterSecurityGroupRules:   masterRules,
 			EtcdELBSecurityGroupName:   key.SecurityGroupName(&cfg.CustomObject, "etcd-elb"),
 			EtcdELBSecurityGroupRules:  getEtcdRules(cfg.CustomObject, cfg.ControlPlaneVPCCidr),
+			VPCID:                      cc.Status.ControlPlane.VPC.ID,
```
Comment (lachlanh) [inline service/controller/resource/tccpn/create.go]: this lines up with the template, not sure if it is still required here though.

File: service/controller/resource/tccpnoutputs/create.go (hunk 1 above)
Comment (lachlanh) [reply service/controller/resource/tccpnoutputs/create.go]: we weren't sure whether it was needed, @xh3b4sd can probably give more context.

**Comment by lachlanh** [comment]
@xh3b4sd can you take a look at the changes so far, was there more that needed to be wired in ?

File: service/controller/resource/tccpnoutputs/create.go (hunk 1 above)
Comment (xh3b4sd) [reply service/controller/resource/tccpnoutputs/create.go]: The initial version was not 100% so we got it straight. This is about the new TCCPN stack and there are only two outputs the stack defines. The rest does not exist so we dropped it.

File: service/controller/resource/tccpn/create.go (hunk 2 above)
Comment (xh3b4sd) [reply service/controller/resource/tccpn/create.go]: I thought this needs the Tenant Cluster VPC ID, not the Control Plane one.

Comment (lachlanh) [reply service/controller/resource/tccpn/create.go]: made the change here, let me know if the pr is good to merge.

**Comment by xh3b4sd** [comment]
LGTM, can't approve my own PR. Go and merge.

**Pull request merged**